### ISSUE TITLE
results on test set of celeba-hq 

### ISSUE THREAD

**Issue by cmyyy** [researchmm/PEN-Net-for-Inpainting#4]
Hi, your results looks great, and i am especially curious about the results on test set of celeba-hq as i am looking for comparision, could you send them to me or share by cloud drive?
Thanks a lot~

**Comment by zengyh1900** [comment]
Sorry for my late reply.
We have updated our codes with released pre-trained models recently. 
You can also find released results for celeba-hq [here](https://drive.google.com/open?id=13xa9Gf4q_tu7B3Wr97udBoTXWrMNkrtl)

**Comment by cmyyy** [comment]
Thanks for your update.
 I can't find the pre-trained model of places2, would you upload it again? 
Thx~
